### PR TITLE
[blog] move scrutinizer to phpstan

### PR DESCRIPTION
**Pull request by TomasVotruba**

### CODE REVIEW

File: source/_posts/2017/2017-01-28-why-I-switched-scrutinizer-for-phpstan-and-you-should-too.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
+---
+layout: post
+title: "Why I Switched Scrutinizer for PHPStan and You Should Too"
+perex: '''
+    I used Scrutinizer for a few years now for code coverage and code quality. Configuration was far complex, some issues appeared and build kept failing. But I really wanted a code quality checker for my open-source projects and this was the best tool there is.
```
Comment (foxycode) [inline source/_posts/2017/2017-01-28-why-I-switched-scrutinizer-for-phpstan-and-you-should-too.md]: Not sure about "this was the best tool there is" construct. What about "this was the best tool available"?

File: source/_posts/2017/2017-01-28-why-I-switched-scrutinizer-for-phpstan-and-you-should-too.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+You can read more about it in post with very true title - [PHPStan: Find Bugs In Your Code Without Writing Tests!](https://medium.com/@ondrejmirtes/phpstan-2939cd0ad0e3)
+
+<img src="https://raw.githubusercontent.com/phpstan/phpstan/master/build/phpstan.gif" alt="PHPStan in action" class="thumbnail">
+
+
+## Why I prefer it over Scrutinizer (and You Should Too)
+
+### It is open-source
+
+**I can improve it, I can add an issue, I can see its development**. That's all I can't do in Scrutinizer. It used to be [open-source](https://github.com/scrutinizer-ci/scrutinizer) but got hidden. That's was a huge step back.
```
Comment (foxycode) [inline source/_posts/2017/2017-01-28-why-I-switched-scrutinizer-for-phpstan-and-you-should-too.md]: "That's" -> "That" in last sentence.

File: source/_posts/2017/2017-01-28-why-I-switched-scrutinizer-for-phpstan-and-you-should-too.md [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+
+### It Focuses just on PHP
+
+**It's PHP tool that checks PHP code.** Scrutinizer, on the other hand, focuses on various languages - Python, Ruby, soon Java and Scala. That's definitelly a good direction, but not it if simple `composer install` command breaks and is not fixed for months.
```
Comment (foxycode) [inline source/_posts/2017/2017-01-28-why-I-switched-scrutinizer-for-phpstan-and-you-should-too.md]: "but not it if simple" -> "but not if simple"

**Comment by TomasVotruba** [comment]
Thanks for review @foxycode

**Comment by TomasVotruba** [comment]
Also thank you @ondrejmirtes for such a nice PR. It's pleasure to read and learn from!

**Pull request merged**